File: chatgpt.js
async function func(){
const { Configuration, OpenAIApi } = require("openai");
const configuration = new Configuration({
  apiKey: "<OpenAI-API-Key>",
});
const openai = new OpenAIApi(configuration);
const response = await openai.createCompletion({
  model:"text-davinci-003",
  prompt: "write code to send http post request in js",
  temperature: 0.5,
  max_tokens: 500,
});
console.log(response.data["choices"][0]["text"]);
}
func();
//test

File: chatgpt.py
import os
import openai

# Load your API key from an environment variable or secret management service
openai.api_key = "<OpenAI-API-Key>"
vcontents = "write code to send https request using c++"
vmodel="text-davinci-003"
response = openai.Completion.create(model=vmodel, prompt=vcontents, temperature=0.5, max_tokens=1000)

print(response["choices"][0]["text"])
